### ISSUE TITLE
Will the bot continue to be updated or have you stopped development?

### ISSUE THREAD

**Issue by MoroseCorpse** [Splamy/TS3AudioBot#902]
I just wanted to ask if there will be more updates in the future or if the project will be discontinued. I'm just wondering, because they usually released many updates in short intervals and now nothing new came for a while. This is not an accusation, it is just a question

**Comment by Flakebi** [comment]
Hi,
I’d say development has paused for the time being, it may resume at some point in the future. That said, bugs will still get fixed.

**Comment by MoroseCorpse** [comment]
Okay, that means new features like the new rights system can still take time to be released? And do you know maybe an approximate timeframe how long it can take until the develop version is ready and becomes master?

**Comment by Flakebi** [comment]
Yes, that will definitely take some time. I don’t know when this will happen.
